db/migrate/20230402090532_remove_unwanted_defult_value_from_users_table.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveUnwantedDefultValueFromUsersTable < ActiveRecord::Migration[7.0]
+  def change
+    change_column :users, :first_name, :string, default: nil
+    change_column :users, :last_name, :string, default: nil
+    change_column :users, :email, :string, default: nil
+    change_column :users, :user_name, :string, default: nil
+    change_column :users, :encrypted_password, :string, default: nil
+    change_column :users, :phone, :string, default: nil
+  
+  end
+end
```

Comment:
You don't need to create a separate migration file for now, I think. As there's nothing in your master branch that would break the DB.

Reply:
Still, do you think default value as "nil" is appropriate for any attributes?
`Hint: it's not mandatory to have default value for all attributes`

Reply:
deleted new migration file and modified old one. 
I set default to nil for removing default from schema . 
thank you.

Reply:
is there any good way to remove default from schema ? Thank you